modules/route53__enum/main.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+import argparse
+from botocore.exceptions import ClientError
+import json
+
+module_info = {
+    # Name of the module (should be the same as the filename).
+    'name': 'route53__enum',
+
+    # Name and any other notes about the author.
+    'author': 'Aaron Rea - Scalesec',
+
+    # Category of the module. Make sure the name matches an existing category.
+    'category': 'ENUM',
+
+    # One liner description of the module functionality. This shows up when a
+    # user searches for modules.
+    'one_liner': 'Enumerates Route53 hosted zones and query loggin congiurations',
+
+    # Full description about what the module does and how it works.
+    'description': 'This module enumerates Route53 hosted zones accross an account and correlates them with query logging configs for later use.',
+
+    # A list of AWS services that the module utilizes during its execution.
+    'services': ['Route53'],
+
+    # For prerequisite modules, try and see if any existing modules return the
+    # data that is required for your module before writing that code yourself;
+    # that way, session data can stay separated and modular.
+    'prerequisite_modules': [],
+
+    # External resources that the module depends on. Valid options are either
+    # a GitHub URL (must end in .git), or a single file URL.
+    'external_dependencies': [],
+
+    # Module arguments to autocomplete when the user hits tab.
+    'arguments_to_autocomplete': ['--get_query_logging_config'],
+}
+
+
+parser = argparse.ArgumentParser(add_help=False, description=module_info['description'])
+
+
+def get_hosted_zones(client):
+    try:
+        hosted_zones = client.list_hosted_zones()['HostedZones']
+    except ClientError as error:
```

Comment:
The 'f' is missing for string format printing the literal "{error}" rather than the boto3 error.